src/BlazorWebFormsComponents/Image.razor.cs

```diff
@@ -0,0 +1,18 @@
+using BlazorWebFormsComponents.Enums;
+using Microsoft.AspNetCore.Components;


```

Comment:
@csharpfritz why this weird symbols appear?!!

Reply:
That's a windows crlf line ending.  You've got mixed line-endings in your files

Reply:
I just copy paste from another file